### DISCUSSION TITLE
UWP project in unoapp-winui-net6 template

### DISCUSSION THREAD

**Discussion by HavenDV** [unoplatform/uno#8108] [General]
I noticed another problem - unoapp-winui-net6 generates a UWP project, which is then not used in the solution.
It has interesting features(I also see it [there](https://platform.uno/blog/announcing-uno-platform-3-0-support-for-winui-3-0-preview-1/)):
```xml
<PropertyGroup>
   <WindowsXamlEnableOverview>true</WindowsXamlEnableOverview>
   <IsWinUIAlpha Condition="'$(IsWinUIAlpha)' == ''">true</IsWinUIAlpha>
   <WindowsKitsPath Condition="'$(IsWinUIAlpha)' == 'true'">WinUI-Alpha-Projects-Don-t-Use-SDK-Xaml-Tools</WindowsKitsPath>
</PropertyGroup>

<ItemGroup>
 <PackageReference Include=""Microsoft.WinUI"">
  <Version>3.0.0-alpha.200210.0</Version>
 </PackageReference>
</ItemGroup>
```

Here comes the question - at some point it became deprecated and you removed it from the solution and forgot to remove it from the template. Or there are some other problems here, so the UWP project was temporarily excluded from the solution, but remained in the template.

I want to help the project in some way, and first of all, I would like the uno templates to be close to ideal in order to cause the least number of problems and questions for beginners. There are still warnings that appear during assembly and an error with the Mobile project, where you need to change the template and explicitly specify the full name of Application when inheriting, but this is later

**Comment by jeromelaban** [comment]
Thanks for the feedback. net6 is still very much in constant changes as winui and net6 mobile are fixing issues.

For the uwp target, it's likely an authoring error as the dotnet new templates are build from the vsix folder structure:
https://github.com/unoplatform/uno/blob/master/src/SolutionTemplate/Uno.ProjectTemplates.Dotnet/content/unoapp-winui-net6/.template.config/template.json#L155-L157
